<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>add_task</class>
 <widget class="QMainWindow" name="add_task">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>634</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>580</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>482</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Görev Ekle</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(239, 255, 254);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>7</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <property name="spacing">
         <number>7</number>
        </property>
        <property name="leftMargin">
         <number>7</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>KONU BAŞLIĞI</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_task_title">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>60</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>60</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Yu Gothic Light</family>
              <pointsize>16</pointsize>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">:hover { background-color: white;}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="maxLength">
             <number>20</number>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="Line" name="line_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="minimumSize">
             <size>
              <width>300</width>
              <height>30</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>300</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>14</pointsize>
             </font>
            </property>
            <property name="text">
             <string>KONU İÇERİĞİ</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="txt_task_detail">
            <property name="minimumSize">
             <size>
              <width>350</width>
              <height>250</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>800</width>
              <height>280</height>
             </size>
            </property>
            <property name="font">
             <font>
              <family>Yu Gothic Light</family>
              <pointsize>16</pointsize>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">:hover { background-color: white;}</string>
            </property>
            <property name="plainText">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0,0,0,0,0,0">
        <property name="spacing">
         <number>7</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>7</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei Light</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Başlangıç Tarihi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="start_time">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="wrapping">
           <bool>false</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="currentSectionIndex">
           <number>0</number>
          </property>
          <property name="date">
           <date>
            <year>2019</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>20</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Microsoft YaHei Light</family>
            <pointsize>10</pointsize>
            <italic>false</italic>
            <underline>true</underline>
           </font>
          </property>
          <property name="text">
           <string>Bitiş Tarihi</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="end_time">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="calendarPopup">
           <bool>true</bool>
          </property>
          <property name="currentSectionIndex">
           <number>0</number>
          </property>
          <property name="date">
           <date>
            <year>2019</year>
            <month>1</month>
            <day>1</day>
           </date>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="important_level">
          <property name="minimumSize">
           <size>
            <width>180</width>
            <height>50</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>180</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <pointsize>13</pointsize>
           </font>
          </property>
          <property name="currentText">
           <string>Çok Önemli</string>
          </property>
          <item>
           <property name="text">
            <string notr="true">Çok Önemli</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Önemli</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Önemsiz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string extracomment="asdfasdf">Çok Önemsiz</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_ekle">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>110</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>160</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Gabriola</family>
            <pointsize>45</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: qlineargradient(spread:reflect, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(175, 255, 255, 255), stop:0.845771 rgba(245, 255, 255, 255));

color: rgb(104, 118, 117);</string>
          </property>
          <property name="text">
           <string>EKLE</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>634</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
